packages/admin/cms-admin/src/index.ts

```diff
@@ -114,6 +114,11 @@ export { useSiteConfig } from "./sitesConfig/useSiteConfig";
 export { useSitesConfig } from "./sitesConfig/useSitesConfig";
 export { AzureAiTranslatorProvider } from "./translation/AzureAiTranslatorProvider";
 export { CurrentUserInterface, CurrentUserProvider, useCurrentUser, useUserPermissionCheck } from "./userPermissions/hooks/currentUser";
+export { UserBasicData } from "./userPermissions/user/basicData/UserBasicData";
+export { ImpersonationButton, StopImpersonationButton } from "./userPermissions/user/ImpersonationButtons";
```

Comment:
It's odd that one is called ImpersonationButton and the other StopImpersonationButton. Could we improve this?